content/ADR-151.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: adr
+adr: 151
+title: Rendering order for UI entities
+date: 2022-12-14
+status: Review
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - menduz
+---
+
+## Abstract
+
+This document describes the requirements to render the UI entities for the Decentraland Protocol. TL;DR; Pre-order using the entity tree defined by the UiTransform ([ADR-124](/adr/ADR-124)).
+
+## Specification
+
+The UI entities are derived from a synthetic tree of entities defined by the UiTransformComponent ([ADR-124](/adr/ADR-124)). That tree should be rendered from the root to the leafs recursively using a pre-order sequencing.
+
+That means the children will overwrite the pixels of the parent. This is a common practice, HTML, ReactNative and Unity's UiToolkit behave this way out of the box.
```

Comment:
I would say again the word entity here to identify that the pixels belong to the entities:
```suggestion
That means that children entities will overwrite the entities' pixels of their parents. This is a common practice, HTML, ReactNative and Unity's UiToolkit behave this way out of the box.
```